historical_system_profiles/config.py

```diff
@@ -36,3 +36,6 @@ def get_namespace():
 aws_region_name = os.getenv("CW_AWS_REGION_NAME", "us-east-1")
 log_group = os.getenv("LOG_GROUP", "platform-dev")
 namespace = get_namespace()
+
+# 7 days in seconds
+valid_profile_age_seconds = int(os.getenv("VALID_PROFILE_AGE_SECONDS", 604800))
```

Comment:
If we are planning to expose this an environment variable I would prefer days and do the math internally. This makes it easier to grok and configure since days is how we think about this feature.

Reply:
I agree re: days in terms of usability. However, seconds makes it easier to test the feature with automation. For example, the test can set the env var, sleep X seconds, confirm a value doesn't appear, then unset.

Reply:
Testing though can send in a fraction of days though right?

Reply:
That is true.

@msager27 would fractional days be OK for setting the expiration, instead of seconds?

Reply:
Yes, fractional would work if the service can accept values like 0.00005787037, which would equate to 5 seconds.